### DISCUSSION TITLE
亞泥&元智&遠傳🏭︎

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#146] [General]
**新聞內容:**
亞泥自 30 年前開始投入水泥生產控制系統的開發工作，採自行設計、製造、維修策略，因此利用過往累績龐大的生產數據資料庫，和元智大學產學合作，導入「人工智慧理論」和強化「**學習及深度學習演算法**」，建置「水泥 4.0」專案。
另一方面，在關係企業遠傳電信協助下，亞泥也布建 IT 基礎設施和**數據分析平台**，並應用在各種 **IoT 物聯網**上，打造智慧工廠。
亞泥分析，自建控制能降低維修困難度和複雜度、相互支援備品、優化設備零件管理，並能藉由遠端支援，提供技術查修、縮短停機時間，避免能源耗損。以亞泥花蓮廠為例，從系統更新優化到節省下的維修工程費用，累計最多可達將近 8000 萬元，邁向綠色生產目標，2050 淨零碳排絕非口號，而且要提前達成。
**新聞心得:**
亞泥與兩家機構的合作，優化了整個工廠運作，而且還使公司節省了龐大費用，喔買尬一舉兩得，而且最近看到更多企業利用人工智慧讓各領域的發展更加智慧化，感覺台灣的科技在企業應用發展有很大的進步。
**新聞來源:**
 https://news.cnyes.com/news/id/4787901?exp=a

![def91e10276eaf9114bffbc8d9cad255](https://user-images.githubusercontent.com/90883526/146633259-f3ebe230-54b4-462e-92bb-aeab6a2e6407.jpg)

**Comment by wh20502** [comment]
感覺傳統產業公司與AI結合，就能夠稍微脫胎換骨一下，不管是節省成本或是開發新功能。
